scripts/bert/finetune_classifier.py

```diff
@@ -512,6 +519,9 @@ def train(metric):
         # inference on dev data
         for segment, dev_data in dev_data_list:
             metric_nm, metric_val = evaluate(dev_data, metric, segment)
+            if metric_val < metric_history[-1][-1] and args.early_stop:
+                should_stop = True
+                logging.info('Early stop at epoch %d', epoch_id)
```

Comment:
Might need to have patience, otherwise it can stop very early due to small fluctuation.

Reply:
What's the best practice? Should we have sth like "worse accuracy in two consecutive epochs"?